mmv1/third_party/terraform/utils/kms_utils.go

```diff
@@ -77,6 +77,11 @@ type kmsCryptoKeyId struct {
 	Name      string
 }
 
+type kmsCryptoKeyVersionId struct {
+	KeyRingId kmsKeyRingId
```

Comment:
The parent resource is a crypto key rather than a key ring.

```suggestion
	cryptoKeyId kmsCryptoKeyId
```